controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -219,6 +232,29 @@ func (r *Reconciler) ReconcileNormal(ctx *pkgctx.VolumeContext) error {
 		}
 	}
 
+	legacyAttachments, err := r.getAttachmentsForVM(ctx)
+	if err != nil {
+		ctx.Logger.Error(err, "Error getting existing CnsNodeVmAttachments for VM")
```

Comment:
nit: wrap in return error?

Reply:
Good catch.  I am 46 minutes in the test run and this is copied over from volume controller v1. So, is it okay if I merge this and we will handle it in a followup?